flake.nix

```diff
@@ -87,6 +87,8 @@
             patches = [ ./package/nix/kevm.patch ];
 
             postPatch = ''
+              substituteInPlace ./Makefile \
+                --replace 'PYK_ACTIVATE := . $(VENV_DIR)/bin/activate' 'PYK_ACTIVATE := true'
```

Comment:
@goodlyrottenapple I think ti would be more natural to pass `PYK_ACTIVATE=true` as a CLI option than to do this substitution like this, but I could not figure out how to do that given this current flake. It seemed that I could possibly do something with `buildFlags`, but that is not used here, and maybe I could also do something with `buildPhase`, but that's also not used here. Any suggestions or is this the best method?